TM1py/Utils/Utils.py

```diff
@@ -742,6 +743,18 @@ def get_dimensions_from_where_clause(mdx: str) -> List[str]:
     return [dimension_name_from_element_unique_name(unique_name) for unique_name in unique_names]
 
 
+def get_cube(mdx: str) -> str:
+    _, sub_mdx = mdx.upper().replace(" ", "").split("FROM[")
+    return sub_mdx[: sub_mdx.find("]")]
+
+
+def resembles_mdx(mdx: str) -> bool:
+    pattern = r"(?s)(?i).*SELECT.*ON.*FROM.*\[.*].*"
+    if re.search(pattern=pattern, string=mdx):
+        return True
+    return False
```

Comment:
@rkvinoth, @scrambldchannel 
Any thoughts or suggestions on this regex? Does it look valid? 
I am using it elsewhere to differentiate between a cube name and an MDX query.

Reply:
@MariusWirtz  I think `r"(?s)(?i).*SELECT.*ON.*FROM"` should be enough. Because we can pass the cube name without `[]` (you know the scenario).

Reply:
@rkvinoth thanks for the hint! I will fix it.